h5py/tests/__init__.py

```diff
@@ -7,35 +7,18 @@
 # License:  Standard 3-clause BSD; see "license.txt" for full license terms
 #           and contributor agreement.
 
-from __future__ import print_function
-
-import sys
-from .common import ut
-
-from . import old, hl
-
-MODULES = old.MODULES + hl.MODULES
-
-
-def mname(obj):
-    """ Get the full dotted name of the test method """
-    mod_name = obj.__class__.__module__.replace('h5py.tests.','')
-    return "%s.%s.%s" % (mod_name, obj.__class__.__name__, obj._testMethodName)
-
-
-def run_tests(verbose=False):
-    """ Run tests with TextTestRunner.  Returns a TestResult instance.
-
-    """
-    suite = ut.TestSuite()
-    for m in MODULES:
-        suite.addTests(ut.defaultTestLoader.loadTestsFromModule(m))
-    result = ut.TextTestRunner(verbosity=1).run(suite)
-
-    if verbose:
-        for (case, reason) in result.skipped:
-            print("S  %s (%s)" % (mname(case), reason), file=sys.stderr)
-        for (case, reason) in result.expectedFailures:
-            print("X  %s" % mname(case), file=sys.stderr)
-
-    return result
+from __future__ import absolute_import
+
+
+def run_tests(args=''):
+    try:
+        from pytest import main
+    except ImportError:
+        print("Tests require pytest, pytest not installed")
```

Comment:
This should probably have a `return 1` so that the exit code indicates something went wrong.

Reply:
Good catch, I've added that now.